### PR TITLE
[fix bug 1363299, 1363300, 1363302] Localize Firefox hub pages

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Updates strings for Firefox hub pages and prepares templates for translation.
~- Should probably wait until after #4904 merges to push out strings, so we can add alt string for  `/firefox/focus/`~
- URLs:
  - `/firefox/`
  - `/firefox/desktop/`
  - `/firefox/android/`
  - `/firefox/ios/`
  - `/firefox/focus/`
  - `/firefox/features/`
  - `/firefox/features/fast/`  
  - `/firefox/features/independent/`
  - `/firefox/features/private-browsing/`
  - `/firefox/features/memory/`
  - `/firefox/features/bookmarks/`
  - `/firefox/features/password-manager/`

~Note: this is still WIP, will update here once strings are final.~

## Bugzilla link
- https://bugzilla.mozilla.org/show_bug.cgi?id=1363299
- https://bugzilla.mozilla.org/show_bug.cgi?id=1363300
- https://bugzilla.mozilla.org/show_bug.cgi?id=1363302

## Testing
- https://www-demo5.allizom.org/firefox/
- https://www-demo5.allizom.org/firefox/desktop/
- https://www-demo5.allizom.org/firefox/android/
- https://www-demo5.allizom.org/firefox/ios/
- https://www-demo5.allizom.org/firefox/focus/
- https://www-demo5.allizom.org/firefox/features/
- https://www-demo5.allizom.org/firefox/features/fast/
- https://www-demo5.allizom.org/firefox/features/private-browsing/
- https://www-demo5.allizom.org/firefox/features/independent/
- https://www-demo5.allizom.org/firefox/features/memory/
- https://www-demo5.allizom.org/firefox/features/bookmarks/
- https://www-demo5.allizom.org/firefox/features/password-manager/

## Checklist
- [x] Requires l10n changes.
- [x] Related functional & integration tests passing.

### CODE REVIEW

**Comment by alexgibson** [comment]
@flodolo @peiying2 - OK, this PR now includes the Focus page too. I believe I have all the copy changes our copywriter suggested, although I'd be more than happy to take any further feedback you have and suggest edits to further help improve l10n.

**Comment by alexgibson** [comment]
I should also add - I'm not sure how you plan to handle the lang files for this, but we can expect many more `/feature/` pages to come in the future.

**Comment by alexgibson** [comment]
There's another page incoming in #4923

**Comment by peiying2** [comment]
@alexgibson thank you for the update. Are all the copies final at this point?

**Comment by alexgibson** [comment]
> @alexgibson thank you for the update. Are all the copies final at this point?

For this pull request yes (unless yourself or Flod have any further suggestions to alter copy)

File: bedrock/firefox/templates/firefox/hub/home.html [hunk 1]
```diff
@@ -41,7 +36,9 @@
         <div class="header-container">
           <div class="header-content">
             <h1 class="page-title"><span>{{ _('Firefox') }}</span></h1>
+            {# L10n: this is the main page heading shown underneath the Firefox logo. #}
             <p class="tagline"><span>{{ _('The 100% fresh, free-range, ethical browser') }}</span></p>
+            {# L10n: here is an alternative page heading should the main heading not translate easily: "Browse like you’re meant to...fast, free and privately." #}
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/hub/home.html]: This doesn't work, the comment is associated to the next string extracted. You should merge it with the comment before.

**Comment by alexgibson** [comment]
> And I have the feeling this is going to be quite a bloodbath :-\

Probably quite the understatement :/

> Given the number of shared strings across templates in /products and /features, we need a shared lang file to add to all of them. I suggest to use firefox/shared.lang.

Makes sense, will do 👍 

> I also really need someone to explain me what "tab hop" means.

I believe "tab hop", or "tab hopping" is being used like the browser equivalent to "channel surf" or "channel surfing" on TV, as an example. Would changing it to "hop tabs" make it any easier?

> Across templates, there are a bunch of hard-coded strings, e.g. in firefox/products/android.html

My error, will fix.

> Can you keep further changes in separate commits until the end, for review sanity?

Can do

**Comment by alexgibson** [comment]
@flodolo added some review fixes in a separate commit, thanks

**Comment by alexgibson** [comment]
I ran this branch against the full cross-browser test suite and it is now passing: https://ci.us-west.moz.works/blue/organizations/jenkins/bedrock_multibranch_pipeline/detail/run-integration-tests/39/pipeline

**Comment by alexgibson** [comment]
@craigcook ok, I think this is ready for review / merge. We're not going to start localizing until after all-hands, but that shouldn't need to block merge.

**Comment by alexgibson** [comment]
Ok, rebased this now that /bookmarks and /password-manager have merged. There are no changes to those files as they already import the shared lang file.

**Comment by flodolo** [comment]
I'm going to extract strings locally to simplify the work for next week. Please let me know it there are any string changes.

**Comment by alexgibson** [comment]
> I'm going to extract strings locally to simplify the work for next week. Please let me know it there are any string changes.

I don't plan to add any more to this PR - thanks.

**Comment by alexgibson** [comment]
Squashed commits

**Pull request merged**